src/components/app/details/cIDetails/CIDetails.tsx

```diff
@@ -1161,6 +1162,12 @@ const SecurityTab: React.FC<{ triggerHistory: History }> = (props) => {
         }
     }, [props.triggerHistory.artifactId])
 
+    const redirectToCreate = () => {
+        const ciPipelineId = props?.triggerHistory?.ciPipelineId
+        const url = `${URLS.APP}/${appId}/${URLS.APP_CONFIG}/${URLS.APP_WORKFLOW_CONFIG}/${ciPipelineId}/${URLS.APP_CI_CONFIG}/${ciPipelineId}/build`
```

Comment:
we can directly use the value in push no need to create variables

Reply:
Done